### PR TITLE
Swift: Make `Callable` a sub type of `AstNode`

### PR DESCRIPTION
**Pull request by hvitved**
The class `Callable` was manually made a sub class of `AstNode` in `swift/ql/lib/codeql/swift/elements/Callable.qll`. This PR adjusts `schema.py` to reflect that.

### CODE REVIEW

File: swift/downgrades/1a24fefd78baf8af6c104d9e63849f3f6e1ef1a3/upgrade.properties
Comment (redsun82) [inline swift/downgrades/1a24fefd78baf8af6c104d9e63849f3f6e1ef1a3/upgrade.properties:1]: let's not worry about upgrade/downgrade scripts, we will start doing those only when we promise stability

**Review by redsun82 — COMMENTED** [review]
looks good, but let's remove upgrade/downgrade scripts, we're not doing them yet (and they are not even checked by CI for that reason)

File: swift/downgrades/1a24fefd78baf8af6c104d9e63849f3f6e1ef1a3/upgrade.properties
Comment (hvitved) [reply swift/downgrades/1a24fefd78baf8af6c104d9e63849f3f6e1ef1a3/upgrade.properties:1]: Note: This is Swift, not Rust.

Comment (redsun82) [reply swift/downgrades/1a24fefd78baf8af6c104d9e63849f3f6e1ef1a3/upgrade.properties:1]: aaaah I think I need a second coffee :sweat_smile:

**Review by redsun82 — APPROVED** [review]

**Pull request merged**